openedx/core/djangoapps/schedules/models.py

```diff
@@ -5,10 +5,18 @@
 
 class Schedule(TimeStampedModel):
     enrollment = models.OneToOneField('student.CourseEnrollment', null=False)
-    active = models.BooleanField(default=True, help_text=_('Indicates if this schedule is actively used'))
-    start = models.DateTimeField(help_text=_('Date this schedule went into effect'))
+    active = models.BooleanField(
+        db_index=True,
```

Comment:
I don't think indexing on `active` is likely to be useful, since the cardinality is so low. Even a compound index isn't likely to save us much over a just an index over time ranges.